tools/stress-cluster/chaos/README.md

```diff
@@ -181,6 +182,34 @@ Fields in `Chart.yaml`
 1. Extra fields in `annotations` can be set arbitrarily, and used via the `-Filters` argument to the [stress test deploy
    script](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/stress-testing/deploy-stress-tests.ps1).
 
+### Stress Test Auth
+
+Stress tests are authenticated via [workload identity for AKS](https://learn.microsoft.com/azure/aks/workload-identity-overview).
+
+To authenticate to Azure, tests can choose to use `DefaultAzureCredential`, `WorkloadIdentityCredential`, or `AzureCliCredential`,
+though it is recommended to use `DefaultAzureCredential` for ease of switching between local machine runs and container runs in the cluster.
+The container that a stress test runs in will have a federated identity token available in its environment that can be used to authenticate.
+This credential is made available automatically by the cluster.
+
+`DefaultAzureCredential` and `WorkloadIdentityCredential` should work automatically with no setup. `AzureCliCredential` can also be used
+provided a login step is added to the test invocation. This method is also useful for test jobs that may need to exercise additional azure
```

Comment:
I think we should just eliminate any mention of AzureCliCredential. A bit later, in this doc, you cover how they could use the Azure CLI - that seems like a reasonable spot to document the possibility.

We definitely don't want them to login to the CLI just to get their normal test running (ie, mentioning it here confuses things).

Reply:
Fair point, and I'm considering not even mentioning `WorkloadIdentityCredential` either. We should just say in code, use DAC, and in a script, use az cli login?

Reply:
I was okay with that, mostly because I it helps in identifying which credential was supposed to work. Went through this a little with @LibbaLawrence where we weren't sure which credential failure was the important one.